src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -218,35 +233,74 @@ tooltip.BloodMagic.telepositionFocus.enhanced=Benutzt, um Blöcke zu verschieben
 tooltip.BloodMagic.telepositionFocus.reinforced=Benutzt, um Blöcke zu verschieben
 tooltip.BloodMagic.telepositionFocus.demonic=Benutzt, um Blöcke zu teleportieren
 
+tooltip.BloodMagic.livingArmour.upgrade.speed=Schnelle Füße
+tooltip.BloodMagic.livingArmour.upgrade.digging=Zergenhafte Stärke
+tooltip.BloodMagic.livingArmour.upgrade.poisonResist=Giftungsresistenz
+tooltip.BloodMagic.livingArmour.upgrade.selfSacrifice=Zähe Handballen
+tooltip.BloodMagic.livingArmour.upgrade.knockback=Body Builder
+tooltip.BloodMagic.livingArmour.upgrade.physicalProtect=Zähe Haut
+tooltip.BloodMagic.livingArmour.upgrade.health=Gesund
+tooltip.BloodMagic.livingArmour.upgrade.meleeDamage=Heftiger Schlag
+tooltip.BloodMagic.livingArmour.upgrade.arrowShot=Trickschuss
+tooltip.BloodMagic.livingArmour.upgrade.level=%s (Level %d)
+
+tooltip.BloodMagic.will=Willensqualität: %1$,.2f
+tooltip.BloodMagic.sentientSword.desc=Benutzt Dämonenwille um volles Potential freizusetzen
+tooltip.BloodMagic.soulGem.petty=Ein Juwel, das eine winzige Menge an Willen aufnehmen kann
+tooltip.BloodMagic.soulGem.lesser=Ein Juwel, das ein wenig Willen festhalten kann
+tooltip.BloodMagic.soulGem.common=Ein Juwel, das eine ordentliche Menge an Willen aufnehmen kann
+tooltip.BloodMagic.soulGem.greater=Ein Juwel, das eine größere Menge an Willen aufnehmen kann
+tooltip.BloodMagic.soulGem.grand=Ein Juwel, das eine große Menge an Willen aufnehmen kann
+tooltip.BloodMagic.soulSnare.desc=Werfe nach einem Monster und töte es anschließend, um seinen Dämonischen Willen zu erhalten
+
 # Ritual
 ritual.BloodMagic.testRitual=Testritual
-ritual.BloodMagic.waterRitual=Ritual des vollen Frühlings
+ritual.BloodMagic.waterRitual=Ritual der vollen Quelle
 ritual.BloodMagic.lavaRitual=Serenade des Nethers
 ritual.BloodMagic.greenGroveRitual=Ritual des grünen Hains
 ritual.BloodMagic.jumpRitual=Ritual des hohen Sprungs
 ritual.BloodMagic.wellOfSufferingRitual=Quelle des Leidens
 ritual.BloodMagic.featheredKnifeRitual=Ritual der gefederten Klinge
 ritual.BloodMagic.regenerationRitual=Ritual der Regeneration
+ritual.BloodMagic.harvestRitual=Ritual der Mondernte
```

Comment:
I'd translate it as "Erntezeit", as "Harvest Moon" is kind of a term for that in English.